pkgs/applications/virtualization/docker/default.nix

```diff
@@ -140,31 +157,19 @@ rec {
       homepage = http://www.docker.com/;
       description = "An open source project to pack, ship and run any application as a lightweight container";
       license = licenses.asl20;
-      maintainers = with maintainers; [ offline tailhook ];
+      maintainers = with maintainers; [ offline tailhook vdemeester ];
       platforms = platforms.linux;
     };
   };
 
-  docker_17_03 = dockerGen rec {
-    version = "17.03.2-ce";
-    rev = "f5ec1e2"; # git commit
-    sha256 = "1y3rkzgg8vpjq61y473lnh0qyc6msl4ixw7ci2p56fyqrhkmhf96";
-    runcRev = "54296cf40ad8143b62dbcaa1d90e520a2136ddfe";
-    runcSha256 = "0ylymx7pi4jmvbqj94j2i8qspy8cpq0m91l6a0xiqlx43yx6qi2m";
-    containerdRev = "4ab9917febca54791c5f071a9d1f404867857fcc";
-    containerdSha256 = "06f2gsx4w9z4wwjhrpafmz6c829wi8p7crj6sya6x9ii50bkn8p6";
-    tiniRev = "949e6facb77383876aeff8a6944dde66b3089574";
-    tiniSha256 = "0zj4kdis1vvc6dwn4gplqna0bs7v6d1y2zc8v80s3zi018inhznw";
-  };
-
-  docker_17_05 = dockerGen rec {
-    version = "17.05.0-ce";
-    rev = "90d35abf7b3535c1c319c872900fbd76374e521c"; # git commit
-    sha256 = "1m4fcawjj14qws57813wjxjwgnrfxgxnnzlj61csklp0s9dhg7df";
-    runcRev = "9c2d8d184e5da67c95d601382adf14862e4f2228";
-    runcSha256 = "131jv8f77pbdlx88ar0zjwdsp0a5v8kydaw0w0cl3i0j3622ydjl";
-    containerdRev = "9048e5e50717ea4497b757314bad98ea3763c145";
-    containerdSha256 = "1r9xhvzzh7md08nqb0rbp5d1rdr7jylb3da954d0267i0kh2iksa";
+  docker_17_06 = dockerGen rec {
+    version = "17.06.0-ce-rc2";
```

Comment:
@NeQuissimus oh right, didn't update that yet.. was trying to figure out the errors before 😛